package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-screen-brightness",
-  "version": "1.0.1",
+  "version": "2.0.0",
```

Comment:
```suggestion
  "version": "2.0.0-alpha",
```

Let's throw this onto alpha versioning. 
With semver prerelease tags, we'll be able to rapidly release without having to manually test and verify each version. That has been the current holdup for project maintainers.